src/store/quoteReplyStore.js

```diff
@@ -39,9 +39,7 @@ const getters = {
 	},
 
 	currentMessageInput: (state) => (token) => {
-		if (state.currentMessageInput[token]) {
-			return state.currentMessageInput[token]
-		}
+		return state.currentMessageInput[token] ?? ''
 	},
```

Comment:
It doesn't break anything, but we should update also `currentMessageInput` usages